lib/galaxy/app.py

```diff
@@ -576,6 +582,25 @@ def __init__(self, configure_logging=True, use_converters=True, use_display_appl
 
         self._configure_tool_shed_registry()
         self._register_singleton(tool_shed_registry.Registry, self.tool_shed_registry)
+        self._register_celery_galaxy_task_components()
+
+    def _register_celery_galaxy_task_components(self):
+        """
+        Register subtype class instance to support implementation of a user rate limit for execution of celery tasks.
+        The default supertype class does not enforce a user rate limit. This is the case if the celery_user_rate_limit
+        config param is default value of 0.0
+        """
+        if self.config.celery_user_rate_limit:
+            force_standard_before_start = self.config.get("celery_user_rate_limit_standard_before_start", False)
+            if not force_standard_before_start and self.config.database_connection.startswith("postgres"):
```

Comment:
```suggestion           
            if not self.config.celery_user_rate_limit_standard_before_start and self.config.database_connection.startswith("postgres"):
```
We don't need to use `get` to access a config value: any value `foo` defined in the schema can be accessed here as `self.config.foo`; if the `foo` option is not set, it'll contain the default value as per its schema definition.
If you'd rather keep `force_standard_before_start` as the variable name, that's fine.